Makefile

```diff
@@ -85,11 +81,19 @@ third-party-try-opt: third-party-try-re2 third-party-try-gmp
 third-party-try-re2: FORCE
 	-@if [ -z "$$CHPL_REGEXP" ]; then \
 	cd third-party && $(MAKE) try-re2; \
+	if [ "llvm" = `${CHPL_MAKE_HOME}/util/chplenv/chpl_llvm.py` ]; then \
+	export CHPL_TARGET_COMPILER=clang-included && \
+	$(MAKE) try-re2; \
+	fi \
 	fi
 
 third-party-try-gmp: FORCE
 	-@if [ -z "$$CHPL_GMP" ]; then \
 	cd third-party && $(MAKE) try-gmp; \
+	if [ "llvm" = `${CHPL_MAKE_HOME}/util/chplenv/chpl_llvm.py` ]; then \
```

Comment:
Should these be `if [ "none" != `${CHPL_MAKE_HOME}/util/chplenv/chpl_llvm.py` ]` ?